packages/forms/src/UIForm/fieldsets/Array/DefaultArrayTemplate.component.test.js

```diff
@@ -97,7 +97,9 @@ describe('Default Array Template component', () => {
 				onReorder={jest.fn()}
 				renderItem={index => <div>Render item {index}</div>}
 				schema={schema}
-				t={(key, options) => options.defaultValue}
+				t={function translate(key, options) {
```

Comment:
I can see the edit of the package.json to remove the errors with i18next-parser with an options to ignore the variables.
I think that it is here your issue with [error] in your console. I prefer use fileFilter option i18next-parser with this file to know what is the file concerned instead of to use.

Reply:
the problem is here when the file is in node-modules (folder where i18n-next extract the keys). the arrow function is transformed by a function named t(key, option). so to fix the problem i transformed the arrow function to a standard function. I should not need to use the option "ignore-variables".